src/NuGet.Core/NuGet.Packaging/PackagesConfigWriter.cs

```diff
@@ -524,10 +525,7 @@ public void WriteFile(string fullPath)
                     @"packages.config.old." + DateTime.Now.ToString("yyyyMMddHHmmss"));
 
                 // Delete configFileCopyPath if it already exists
-                if (File.Exists(configFileCopyPath))
```

Comment:
Wouldn't we still want to check for file exists, to avoid exceptions, which aren't as efficient perf wise?
If so, we could put in FileUtility.Delete or leave in the calling code.